gtsfm/frontend/verifier/opencv_verifier_base.py

```diff
@@ -0,0 +1,145 @@
+"""
+Base-class for OpenCV verifier implementations.
+
+Authors: John Lambert
+"""
+
+import abc
+from typing import Optional, Tuple
+
+import numpy as np
+from gtsam import Cal3Bundler, Rot3, Unit3
+
+import gtsfm.utils.features as feature_utils
+import gtsfm.utils.logger as logger_utils
+import gtsfm.utils.verification as verification_utils
+from gtsfm.common.keypoints import Keypoints
+from gtsfm.frontend.verifier.verifier_base import NUM_MATCHES_REQ_E_MATRIX, NUM_MATCHES_REQ_F_MATRIX, VerifierBase
+
+logger = logger_utils.get_logger()
+
+
+class OpencvVerifierBase(VerifierBase):
```

Comment:
I didn't get the point of this class to be just OpenCV specific? Do we want these 2 functions for every verifier? If yes, then lets make the APIs part of the base verifier. Otherwise lets use the if-else in the verify function itself?